src/libs/theme.js

```diff
@@ -119,7 +121,9 @@ export const Styles = {
       backgroundColor: "#f3f6f7",
       display: "flex",
       justifyContent: "left",
-      height: "51px"
+      height: "51px",
+      borderTopLeftRadius: '10px',
```

Comment:
Instead of defining the pixel value, you can use the term ```inherit```. This way the radius value will be whatever is defined on the parent, which in this case should be the Container div (where the border is drawn).